packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -73,28 +76,45 @@ const CompleteResetPasswordContainer = ({
 
   const isResetWithoutRecoveryKey = !!(code && token);
 
+  const localizedSuccessMessage = ftlMsgResolver.getMsg(
+    'reset-password-complete-header',
+    'Your password has been reset'
+  );
+
   const handleNavigationWithRecoveryKey = () => {
-    navigate('/reset_password_with_recovery_key_verified');
+    navigateWithQuery('/reset_password_with_recovery_key_verified');
   };
 
   const handleNavigationWithoutRecoveryKey = async (
     accountResetData: AccountResetData
   ) => {
-    if (
-      accountResetData.verified &&
-      (isWebIntegration(integration) || integration.isSync())
-    ) {
-      alertBar.success(
-        ftlMsgResolver.getMsg(
-          'reset-password-complete-header',
-          'Your password has been reset'
-        )
-      );
-      return navigate(SETTINGS_PATH, { replace: true });
+    if (accountResetData.verified) {
+      if (isWebIntegration(integration)) {
+        alertBar.success(localizedSuccessMessage);
+        return navigateWithQuery(SETTINGS_PATH, { replace: true });
+      }
+
+      if (integration.isSync()) {
+        alertBar.success(localizedSuccessMessage);
+        // Go to settings, users will see an account recovery key promotion banner
+        return navigateWithQuery(SETTINGS_PATH, { replace: true });
+      }
```

Comment:
nit: I don't see a difference between the two `if`s blocks above.  any reason you didn't want to combine them?

Reply:
Ah yes, these were separate because I was uncertain about the post-reset destination for sync. I could combine the /settings path as default for now. I've also posted in the ux channel to confirm the end point for this flow.